src/plugins/intel_gpu/include/intel_gpu/graph/program.hpp

```diff
@@ -162,6 +162,7 @@ struct program {
     bool is_internal_program() const { return is_internal; }
     const nodes_ordering& get_processing_order() const;
     nodes_ordering& get_processing_order();
+    const std::vector<primitive_id>& get_allocating_order(bool forced_update = false);
```

Comment:
I cannot find calling this function with forced_update = true. Am I missing something?

Reply:
No, it's not used anywhere yet. I made this in case someone needed it.